src/h_vialib/client.py

```diff
@@ -100,8 +103,12 @@ def _url_for(self, doc, query):
         if self._service_url is None:
             raise ValueError("Cannot rewrite URLs without a service URL")
 
-        # Optimisation to skip routing for documents we know are PDFs
-        path = "/pdf" if doc.is_pdf else "/route"
+        # Optimisation to skip routing for documents we know the type of
+        content_type_paths = {
+            ContentType.PDF: "/pdf",
+            ContentType.VIDEO: "/video",
```

Comment:
This endpoint doesn't exist yet.

Reply:
Ok, so here we've both refactored the code to use a dict instead of an `if` / `else` and also added in the `ContentType.VIDEO` case